Dockerfile

```diff
@@ -9,17 +9,16 @@ WORKDIR /build-ocrd
 COPY setup.py .
 COPY requirements.txt .
 COPY requirements_test.txt .
+COPY ocrd_tesserocr ./ocrd_tesserocr
+COPY Makefile .
 RUN apt-get update && \
     apt-get -y install --no-install-recommends \
-    ca-certificates \
-    make \
-    git
-COPY Makefile .
-RUN make deps-ubuntu
-COPY ocrd_tesserocr ./ocrd_tesserocr
+    libtesseract-dev \
```

Comment:
Does this install 4.1 (current stable release)? If it still installs 4.0 (no longer supported, known bugs), that should be changed.

The `ENV` statements in lines 5 and 6 were needed for Tesseract 4.0. Tesseract 4.1 does not require them.

Reply:
Yes, this will install [4.1.0-1](https://www.ubuntuupdates.org/package/core/eoan/universe/base/tesseract)